### PR TITLE
[Global] Minor All-Around Cleanup

### PR DESCRIPTION
**Pull request by rouing**
Doing Cleanups

Draft PR, Do Not Merge

### CODE REVIEW

File: src/modules/gps/gps_tracker.h [hunk 1]
```diff
@@ -6,8 +6,8 @@
  * @date 2024-11-20
  */
 
-#ifndef GPS_TRACKER_H
-#define GPS_TRACKER_H
+#ifndef __GPS_TRACKER_H
```
Comment (rennancockles) [inline src/modules/gps/gps_tracker.h]: I think you forgot the final underscores here

File: .github/workflows/buil_parallel.yml
Comment (bmorcelli) [inline .github/workflows/buil_parallel.yml:1]: This is not needed.

Patch is being applied directly here:
```
platform = https://github.com/bmorcelli/platform-espressif32/releases/download/0.0.4/platform-espressif32.zip
```

File: src/modules/gps/gps_tracker.h (hunk 1 above)
Comment (rouing) [reply src/modules/gps/gps_tracker.h]: Yes I did

File: patch.py
Comment (bmorcelli) [inline patch.py:1]: This patch is removing the file and not replacing for the original..

This patch had been applied in the cfw.. no need to repatch here and in the actions file

Comment (rouing) [reply patch.py:1]: I need to fix it for the Github actions but it does work. If I cant ill toss it

Comment (rouing) [reply patch.py:1]: I got it working

Comment (rouing) [reply patch.py:1]: I had the tools name suffixed by ".exe"

File: .github/workflows/PR_check.yml [hunk 2]
```diff
@@ -23,35 +23,42 @@
           matrix:
             board:
               - {
-                  name: "ESP32-S3 - M5Cardputer",
+                  name: "M5 - Cardputer",
```
Comment (bmorcelli) [inline .github/workflows/PR_check.yml:26]: These modifications will breake the web flasher.. dont touch it, please!

**Review by rouing — COMMENTED** [review]
Checkpoint - Successful and usable.

**Comment by rouing** [comment]
Still needs some things done but my other things rely on this restructure to happen first

**Comment by rouing** [comment]
Rebased on #590

**Pull request closed without merging**